integrations/tensorflow/e2e/mobile_bert_squad_test.py

```diff
@@ -0,0 +1,91 @@
+# Lint as: python3
+"""Test MobileBERT.
+
+Model topology and weights are from
+https://github.com/google-research/google-research/tree/master/mobilebert
+"""
+
+import os
+import posixpath
+
+from absl import app
+from absl import flags
+import numpy as np
+from pyiree.tf.support import tf_test_utils
+import tensorflow.compat.v2 as tf
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_boolean("use_quantized_weights", False,
+                     "Whether to use quantized or floating point weights.")
+
+MAX_SEQ_LENGTH = 384  # Max input sequence length used in mobilebert_squad.
+
+FILE_NAME = "mobilebert_squad_savedmodels.tar.gz"
+MODEL_URL = posixpath.join(
+    "https://storage.googleapis.com/cloud-tpu-checkpoints/mobilebert/",
+    FILE_NAME)
+
+
+class MobileBertSquad(tf.Module):
+  """Wrapper of MobileBertSquad saved model v1."""
+
+  def __init__(self):
+    self.model_path = self.get_model_path()
+    self.saved_model = tf.saved_model.load(self.model_path, tags=["serve"])
+    self.inference_func = self.saved_model.signatures["serving_default"]
+
+  @staticmethod
+  def get_model_path():
+    model_type = "quant_saved_model" if FLAGS.use_quantized_weights else "float"
+
+    # Get_file will download the model weights from a publicly available folder,
+    # save them to cache_dir=~/.keras/datasets/ and return a path to them.
+    model_path = tf.keras.utils.get_file(FILE_NAME, MODEL_URL, untar=True)
+    model_dir = os.path.dirname(model_path)
+    extracted_name = FILE_NAME.split(".")[0]
+    model_path = os.path.join(model_dir, extracted_name, model_type)
+    return model_path
+
+  @staticmethod
+  def get_tflite_v1_kwargs():
```

Comment:
why is this needed? Shouldn't we be able to export the "predict" method to TFLite directly?

Reply:
Unfortunately the model gives error like this when you try to convert it using TFLite in TensorFlow v2:

```python
ValueError: Input 3 of node StatefulPartitionedCall was passed int64 from 41590:0 incompatible with expected resource.
```

The source repository for the model uses `tf.v1.compat`, so Oleg and I decided to just support that. TFLite V1 doesn't have `from_concrete_functions`, which is why I fall back on using `SavedModel` here.

Reply:
Ok. Let's call the method `get_legacy_tflite_saved_model_converter_kwargs`. And document it in README.md

Reply:
Also, in the readme, can you clarly indicate that this should only be used if the normal TFLite path doesn't work.

Reply:
Done!